src/pages/components/accordion/accessibility.mdx

```diff
@@ -0,0 +1,382 @@
+accessibility issues are resolved.
+
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>How it works</AnchorLink>
+  <AnchorLink>Accessibility Considerations</AnchorLink>
+  <AnchorLink>Resources</AnchorLink>
+  <AnchorLink>Accessibility Testing</AnchorLink>
+</AnchorLinks>
+
+## How it works
+
+The Accordion Carbon Component provides a vertically stacked layout for web content, commonly used to reduce scrolling. The accordion header is a button that is used to expand and collapse each accordion panel. Buttons are used so that the accordions are tab-able by keyboard users and accessible to screen readers. Tab key and Shift-Tab keys are used to navigate through each accordion header and all focusable elements in the accordion should be included in the page Tab sequence. The Enter or Space key expand and collapse each accordion panel.
+
+WAI-ARIA roles and states are used to ensure the component is accessible to people using assistive technologies. The accordion component is assigned the ARIA `role="presentation"` because it is used to layout web content. When an accordion panel is collapsed the content is hidden so the ARIA state, `aria-expanded="false"` advises users of assistive technologies that the accordion panel is not visible. When the accordion header is expanded the ARIA state changes to `aria-expanded="true"` and the accordion panel content is displayed. Each accordion header contains an ARIA label, and the header buttons have a aria-control property set that points to the unique id of the panel it controls. When focus is on the Accordion header there is a prominent style change to the border as well as background of the accordion header.
+
+## Accessibility Considerations
+
+This component has been validated to meet the [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) and [Section 508](http://www.section508.gov/) accessibility guidelines, however changes made by the content owner can affect accessibility compliance. Be sure to review and follow the guidance in this section when updating or adding new content to this component.
+
+1. Each accordion header must have a unique title (implemented via the label for the button) that clearly describes the accordion panel content. This is particularly helpful for users of assistive technologies so they have the necessary information to efficiently navigate the content without having to expand every section.
+1. Carbon Components should be used to create the content that displays within each accordion panel.
+1. Avoid keyboard traps when adding components to the accordion panel. For example, the user expands an accordion, but is unable to tab to the next focusable element.
+1. Although the accordion component passes accessibility testing, content authors need to ensure the content that is added to the accordion is accessible. For example, if you add an image to the accordion header or panel you need to include alternative text to pass accessibility testing.
```

Comment:
should this be # 1 - 4

Reply:
It could be, but actually doesn't matter, it will get converted to `<ol>` no matter what the numbers actually are.